src/chainTypes/harvesterTypes.json

```diff
@@ -96,5 +96,6 @@
   "AuthorityList": "Vec<NextAuthority>",
   "ElectionCompute": {
     "_enum": ["OnChain", "Signed", "Authority"]
-  }
+  },
+  "DispatchErrorModule": "DispatchErrorModuleU8"
```

Comment:
Adding this made it pass the block, which kept on failing the previous time. ([#6918](https://polymesh-testnet.subscan.io/block/0x633b8e81e111b8e9353c99adef52c2def3224b69a2e47280ba34c15553b1cb4e?tab=extrinsic) in testnet)